lib/bump.rb

```diff
@@ -39,11 +40,18 @@ def run(bump, options = {})
           bump_set(options[:version], options)
         when "current"
           ["Current version: #{current}", 0]
+        when "show-next"
+          raise InvalidIncrementError unless BUMPS.include?(options[:increment])
+
+          increment = options[:increment]
+          ["Next #{increment} version: #{next_version(current, increment)}", 0]
```

Comment:
cleaner to just return `next_version`, so result can easily be used from other scripts